lib/update_remote_id.rb

```diff
@@ -0,0 +1,51 @@
+# Update events remote_id from existing schedule xml data
```

Comment:
But if you delete this file, how should this mapping work for later conferences? Why not fix it the right way now and not have to think about it again next year?

No one of us will know how to run these tasks again next year and no one will maintain them…